enterprise/backup/src/main/java/org/neo4j/backup/impl/OnlineBackupCommandBuilder.java

```diff
@@ -64,6 +64,13 @@
     private Boolean consistencyCheckLabel;
     private Boolean consistencyCheckOwners;
     private OutputStream output;
+    private Optional<String[]> rawArgs = Optional.empty();
+
+    public OnlineBackupCommandBuilder withRawArgs( String... args )
+    {
+        rawArgs = Optional.of( args );
```

Comment:
This will always be present

Reply:
Yes, but it won't always be called. When it isn't called then `rawArgs = Optional.empty()`